RATapi/utils/enums.py

```diff
@@ -51,11 +53,18 @@ class Strategies(RATEnum):
 
     Random = "random"
     LocalToBest = "local to best"
-    BestWithJitter = "best w/ jitter"
+    BestWithJitter = "best jitter"
     RandomWithPerVectorDither = "vector dither"
     RandomWithPerGenerationDither = "generation dither"
     RandomEitherOrAlgorithm = "either or"
 
+    @classmethod
+    def _missing_(cls, value: Union[int, str]):
+        # legacy compatibility with strategies being 1-indexed ints under the hood
+        if isinstance(value, int):
+            return list(cls)[value - 1]
+        return super()._missing_(value)
```

Comment:
This does the job well, but could you please take a look at the case where the input is 0. What happens is the final value is returned (since we return the -1'th element), but an index of 0 returning the last item is not expected behaviour in any language. What do you think is the right behaviour in this case, and for negative indices?

Reply:
i suppose what we're *really* doing is creating a correspondence between the numbers 1 through 6 and the strategies... i would suggest we just throw an error if the value is less than 1 or greater than 6?

Reply:
Works for me. It already throws an index error for >6, so why not add one for <=0.